app/Main.hs

```diff
@@ -56,7 +51,7 @@ main = do
         V1 -> V1.tzbtcDoc
         V2 -> V2.tzbtcDoc
       in maybe printTextLn writeFileUtf8 mbFilePath
-        (contractDocToMarkdown $ buildLorentzDocWithGitRev gitRev contractDoc)
+        (contractDocToMarkdown $ buildDoc . attachDocCommons gitRev $ contractDoc)
```

Comment:
What's the gist of this change?

Reply:
`buildLorentzDocWithGitRev` is deprecated, the suggested way to update it is to use `buildDoc . attachDocCommons gitRev`, see [this piece of code](https://gitlab.com/morley-framework/morley/-/blob/45d089930010751c1f8389f24c2ffff703e23794/code/lorentz/src/Lorentz/Doc.hs#L151)